forgerock-openbanking-uk-aspsp-as/forgerock-openbanking-uk-aspsp-as-gateway/forgerock-openbanking-uk-aspsp-as-gateway-server/src/main/java/com/forgerock/openbanking/aspsp/as/service/TppRegistrationService.java

```diff
@@ -500,4 +417,16 @@ public OIDCRegistrationResponse getOIDCClient(String token, Tpp tpp) {
         log.debug("Read TPP {} in AM", tpp.getClientId());
         return amoidcRegistrationService.getOIDCClient(token, tpp.getClientId());
     }
+
+    public String getStringClaim(JWTClaimsSet claimSet, String claimName) throws DynamicClientRegistrationException {
+        try {
+            return claimSet.getStringClaim(claimName);
+        } catch (ParseException pe) {
+            String errorString = "Could not parse the " + claimName + " claim from the software " +
+                    "statement provided in the registration request JWT.";
+            log.debug("getStringClaim() {}, Error was ", errorString, pe);
```

Comment:
Could be better log.error instead of log.debug? because contains the errorMessage and exception.